client/src/views/page-components/ProductDetailPage/ProductDetail/Carousel/CarouselCount/CarouselCount.js

```diff
@@ -0,0 +1,26 @@
+export default function CarouselCount({ $target, initialState }) {
+  /**
+   * {
+   *  total : 7,
+   *  current : 1,
+   * }
+   */
+  this.state = initialState;
```

Comment:
좀더 주석이 명확했으면 좋겠네요~
취향차이긴 합니다 🤣

```suggestion
export default function CarouselCount({ $target, initialState }) {
  /**
   * state가 가지는 값은 다음과 같습니다.
   * - total : Carousel이 가지고 있는 총 사진의 개수입니다.
   * - current : 현재 Carousel이 보여주고 있는 index 입니다.
   */
  this.state = initialState;
```

Reply:
주석에 설명을 추가시키니 보기가 좋네요!
명확하게 정의하도록 할게요!! 감사합니다 :)